### DISCUSSION TITLE
Follow discussion by RSS.

### DISCUSSION THREAD

**Discussion by BobSquarePants** [pallets/flask#4961] [Q&A] [answered]
Hi everyone,

As Github is[ not proving RSS to follow conversations](https://github.com/community/community/discussions/31) (and most presumably will never (not their interest))

It might be interesting to found an alternative to GitHub for the discussion (Q & A)
I wanted in my spare time to help people. And RSS is perfect for this.

Maybe Pallets might open a traditional forum, just for the Q&A ? because it often come with an RSS support. https://github.com/flaskbb/flaskbb ? (Didn't tried it yet)

Cheers

**Comment by tachyondecay** [comment, marked as answer]
You can join us [on Discord](https://discord.gg/pallets), or if that isn’t your flavour, #pallets on Libera.Chat is bridged to the Discord's questions channel.

I think there’s a Flask subreddit if forum-like conversations are more your thing.

**Reply by BobSquarePants** [reply]
Thanks @tachyondecay ,

Yeah Discord is more like an [IM ](https://en.wikipedia.org/wiki/Instant_messaging)style and therefor not the best suited to follow open discussion (Thread style).

For about Reddit, I quit it, too much censorship over there :scream: 

I think I will stick here (so far) not the best as it's also not giving pallets folks sovereignty...

**Comment by Thomas-AA** [comment]
Talking about Discord.com I can't join you there for my part ->
![2023-03-04-1631_Discord](https://user-images.githubusercontent.com/118596529/222915107-9ae42cbb-50b5-43bb-9b71-f68f454b5a8d.png)
